### PR TITLE
create pre-commit image

### PR DESCRIPTION
**Pull request by KPostOffice**
Create image for running pre-commit CI task

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
great work @KPostOffice 
please respect the end of line on the files.
and can you also create one based on ubi.

File: thoth-precommit/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Thoth Python 3.6-fedora28 Pre-Commit" \
+      io.openshift.tags="python,python36,pre-commit" \
+      name="thoth-station/thoth-pre-commit:fedora28"
+
+USER 0
+COPY ./ /tmp/src-thoth-coala
```
Comment (harshad16) [inline thoth-precommit/Dockerfile]: can we keep using pre-commit for consistency everywhere. WDYT?

**Comment by goern** [comment]
> please respect the end of line on the files.

that's something pre-commit should handle on localhost before the stuff is pushed to github :p

**Comment by KPostOffice** [comment]
@harshad16 PTAL :smile:

File: thoth-precommit-ubi8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM quay.io/thoth-station/s2i-thoth-ubi8-python36:v0.11.3
+
+ENV SUMMARY="Thoth's toolchain for running pre-commit hooks." \
+    DESCRIPTION="Thoth's toolchain for running pre-commit hooks."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Thoth Python 3.6-fedora28 Pre-Commit" \
```
Comment (harshad16) [inline thoth-precommit-ubi8/Dockerfile]:
```suggestion
      io.k8s.display-name="Thoth Python 3.6-ubi8 Pre-Commit" \
```

File: thoth-precommit-ubi8/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      io.openshift.tags="python,python36,pre-commit" \
+      name="thoth-station/thoth-pre-commit:fedora28"
```
Comment (harshad16) [inline thoth-precommit-ubi8/Dockerfile]:
```suggestion
      name="thoth-station/thoth-pre-commit:ubi8"
```

File: thoth-precommit-ubi8/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+USER 0
+COPY ./ /tmp/src-thoth-precommit
+RUN /usr/bin/pip3 install --requirement /tmp/src-thoth-pylint/requirements.txt
```
Comment (harshad16) [inline thoth-precommit-ubi8/Dockerfile]:
```suggestion
RUN /usr/bin/pip3 install --requirement /tmp/src-thoth-precommit/requirements.txt
```

**Review by harshad16 — APPROVED** [review]
Nice work. 
thanks :)

**Pull request merged**